src/api/data.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+const DATA_URL = 'https://mockurl.com';
+
+export function getRestaurantData(params) {
+  return axios.get(DATA_URL, { params });
+}
+
+export function makeOrderRequest(payload) {
+  return axios.post(DATA_URL, { payload });
+}
```

Comment:
Not a real issue with this just don't understand if this bit of code is being used somewhere or if it's just axios setup for when we have a URL to request data from?

Reply:
It's not being used anywhere just yet. It's mostly a template now.